### PR TITLE
[KYUUBI #5484] Remove legacy Web UI

### PR DESCRIPTION
**Pull request by pan3793**
### _Why are the changes needed?_

Close https://github.com/apache/kyuubi/issues/5484

Kyuubi provides a basic new Web UI which is built on top of Vue3, we can remove the legacy dummy Web UI in 1.8.

The new UI hosts at `http://<host>:<port>/ui/` and the legacy UI hosts at `http://<host>:<port>/`, we should 

1. Remove the legacy UI routing from Jetty
2. Remove all files related to legacy UI
3. Redirect `http://<host>:<port>/` to `http://<host>:<port>/ui/`

### _How was this patch tested?_
- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [x] Add screenshots for manual tests if appropriate

building with the command `build/dist --web-ui`, then `cd dist` and perform `bin/kyuubi run`

access http://0.0.0.0:10099 could correctly redirect to http://0.0.0.0:10099/ui/

image

swagger is render correctly too.
image

- [ ] [Run test](https://kyuubi.readthedocs.io/en/master/contributing/code/testing.html#running-tests) locally before make a pull request

### _Was this patch authored or co-authored using generative AI tooling?_

No.

### CODE REVIEW

**Review by turboFei — APPROVED** [review]

**Comment by link3280** [comment]
LGTM.

**Review by link3280 — APPROVED** [review]

File: kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiRestFrontendService.scala [hunk 1]
```diff
@@ -98,19 +98,13 @@ class KyuubiRestFrontendService(override val serverable: Serverable)
     val proxyHandler = ApiRootResource.getEngineUIProxyHandler(this)
     server.addHandler(authenticationFactory.httpHandlerWrapperFactory.wrapHandler(proxyHandler))
 
-    server.addStaticHandler("org/apache/kyuubi/ui/static", "/static/")
-    server.addRedirectHandler("/", "/static/")
-    server.addRedirectHandler("/static", "/static/")
-    server.addStaticHandler("META-INF/resources/webjars/swagger-ui/4.9.1/", "/swagger-static/")
```
Comment (cfmcgrady) [inline kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiRestFrontendService.scala:104]: it's safety to remove dep `swagger-ui`

https://github.com/apache/kyuubi/blob/5cff4fb98cdc8d8a8d9a1c289a17a17a52a787e5/pom.xml#L836-L846

Comment (pan3793) [reply kyuubi-server/src/main/scala/org/apache/kyuubi/server/KyuubiRestFrontendService.scala:104]: thanks for catching, removed

**Review by cfmcgrady — APPROVED** [review]

**Comment by yaooqinn** [comment]
Thank you all, merged to master and 1.8

**Pull request closed without merging**